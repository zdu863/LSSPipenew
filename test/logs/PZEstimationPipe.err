/global/homes/z/zdu863/.conda/envs/py36/lib/python3.6/site-packages/ceci-0.0.7-py3.6.egg/ceci/stage.py:209: YAMLLoadWarning: calling yaml.load() without Loader=... is deprecated, as the default Loader is unsafe. Please read https://msg.pyyaml.org/load for full details.
  overall_config = yaml.load(open(self.get_input('config')))
