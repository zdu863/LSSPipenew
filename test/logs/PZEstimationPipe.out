Executing stage: PZEstimationPipe
    PZEstimationPipe reading from ./test/inputs/dm.txt
    PZEstimationPipe reading from ./test/inputs/fiducial_cosmology.txt
    PZEstimationPipe writing to ./test/outputs/photoz_pdfs.txt
